### ISSUE TITLE
Remove double updateBadnik

### ISSUE THREAD

**Issue by chipviled** [Nibelion/Sonic_REALMZ#15]
Try this in server:

```
        if ( distance( b.x, b.y, pl.x, pl.y-16 ) < 800 ){
            if (b.a == false) { 
                io.emit("updateBadnik",{ id: i, a: b.a }) 
            }else{
                io.emit("updateBadnik",{
                    id: i,
                    x: b.x,
                    y: b.y,
                    i: b.i,
                    HP: parseInt(b.HP),
                    a: b.a
                });
            }
        };
```

**Comment by xyngraph** [comment]
Ok, which line and which file?
I'll try figuring it out by myself for now

**Comment by xyngraph** [comment]
Okay it's resolved in last branch.

Now it is:

``` javascript
            if ( distance( b.x, b.y, pl.x, pl.y - 16 ) < 800 ){
                if ( b.a == false ) {
                    pl.socket.emit("updateBadnik",{ id: i, a: b.a }) 
                } else {
                    pl.socket.emit("updateBadnik",{
                        id: i,
                        x: b.x,
                        y: b.y,
                        i: b.i,
                        HP: parseInt(b.HP),
                        a: b.a
                    });
                };
            };
```

io.emit sent data to ALL players - possible traffic leak fixed.

**Comment by xyngraph** [comment]
Fixed in #18
